WebRoot/js/zimbraAdmin/domains/model/ZaDomain.js

```diff
@@ -741,6 +744,17 @@ function(tmpObj, newDomain) {
 		attr.setAttribute("n", ZaDomain.A_zimbraFeatureCalendarReminderDeviceEmailEnabled);
 	}
 
+	if(ZaDomain.extendedAttrs.length>0) {
```

Comment:
It can be removed because the length is checked in the next`for` statement.

Reply:
Removed this check.